src/engine/utils/create-patch.ts

```diff
@@ -1,17 +1,13 @@
 import { Entry } from 'contentful'
-import { create as createDiffer, Delta, formatters as diffFormatters, Patch } from '@contentful/jsondiffpatch'
-
-const format: (delta: Delta | undefined) => Patch = diffFormatters.jsonpatch.format
-
-const entryDiff = createDiffer({
-  propertyFilter: function (name: string) {
-    return !['sys'].includes(name)
-  },
-  textDiff: {
-    minLength: Number.MAX_SAFE_INTEGER,
-  },
-})
+import { generateJSONPatch } from 'generate-json-patch'
 
 export const createPatch = ({ targetEntry, sourceEntry }: { targetEntry: Entry<any>; sourceEntry: Entry<any> }) => {
-  return format(entryDiff.diff(targetEntry, sourceEntry))
+  // Temp fix until the types from generate-json-patch are fixed
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  return generateJSONPatch(targetEntry, sourceEntry, {
+    propertyFilter: function (name: string) {
+      return !['sys'].includes(name)
```

Comment:
In changeset-create we had the filter like this:
```
const { length } = pathInfo(context.path)
      // ignore sys prop on root level
      if (length === 1) {
        return !['sys'].includes(name)
      }
      return true
```

Would you say that would be safer here?

Reply:
Great catch @andreascful 🙏